src/linker/Linker/UnconditionalSuppressMessageAttributeState.cs

```diff
@@ -130,7 +169,10 @@ bool IsSuppressedOnElement (int id, ICustomAttributeProvider? provider, out Supp
 					AddSuppression (suppressionInfo, member);
 			}
 
-			return _suppressions.TryGetValue (provider, out suppressions) && suppressions.TryGetValue (id, out info);
+			if (_suppressions.TryGetValue (provider, out suppressions))
+				return true;
+
+			return false;
```

Comment:
nit: you could leave only
```suggestion
			return _suppressions.TryGetValue (provider, out suppressions);
```